project_usability_issue/README.rst

```diff
@@ -0,0 +1,70 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=======================
+Project Usability Issue
+=======================
+
+Several improovements to project:
+#. Add to project issue to search by "Days Since Creation".
```

Comment:
Este ya venía mal, aprovechemos  a mejorar por 

Several improvements:
#. Allow to search by "Days Since Creation" on issues.